app/components/in-view.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
I removed it for now, until I dig more into debugging. Even on my phone, UI is refreshed very quickly compared to the search time on mopidy.
Is it behaving the same for you on all browsers?